assets/sass/components/_frontpage-cards.scss

```diff
@@ -0,0 +1,128 @@
+.frontpage {
+	&.frontpage-section {
+		margin: 1%;
+		margin-top: 30px;
+		> .section-heading {
+			border-bottom: solid 1px #eee;
+			display: flex;
+			justify-content: space-between;
+			padding-left: 10px;
+			padding-right: 10px;
+			width: 100%;
+			> .action {
+				display: flex;
+				flex-direction: row;
+				justify-content: center;
+				align-items: center;
+				font-size: 18px;
+				margin-top: 20px;
+				margin-bottom: 10px;
+			}
+		}
+	}
+	.panel-default-text {
```

Comment:
I was thinking the style guide, but should there be empty line between `}` and the next style selector? Would it help readability?

Same with the nested stuff like

```scss
.foo {
  color: black;
  > .bar {
    color: white;
  }
}
// vs.
.foo {
  color: black;

  > .bar {
    color: white;
  }
}
```

Wanna share some opinions?

Reply:
I don't have much preference. 

At the moment, I've used the SCSS defaults that ship with [Prettier](https://prettier.io/docs/en/why-prettier.html) in vscode, because I like not having to think too much about the styles - if we're going to make some rules, it would be nice to find rules which can be applied automatically like this.

Reply:
For sure, there should be configurations for different formatters and linters. My idea was to write down them first as text and developers could configure their preferred tools and contribute the configs to the repo.

I didn't find the default configs to look for those rules. I guess what ever formatting is fine for now.

Btw. `options.editorconfig` might be handy for the indentation.

Reply:
I guess I mis-described it, [Prettier isn't a linter](https://prettier.io/docs/en/comparison.html) with a list of style rules, as such. It reprints the code consistently - we couldn't, for example, just tweak a setting to get the blank lines.

If we want those, I can switch to a tool like stylelint. Since we didn't have any guidelines yet, I preferred going with the simpler Prettier approach. 

With relatively limited time this summer, I wouldn't caution against putting much time into writing guidelines yet if it's going to block things. With the tools available, files can easily be re-formatted if we get such guidelines in the future.

Reply:
Yes, prettier / auto formatter / beautifier is not a linter, but both families of tools often have configuration.

You can edit configuration, if you like, but I think it's not required. I would prefer to use as much defaults as possible, but I personally prefer there to be a bit of extra space (as empty lines).

Nevertheless, this PR can be accept as it is now.